### ISSUE TITLE
(DBeaver v2.4.5) Oracle Dates do not display time portion correctly

### ISSUE THREAD

**Issue by leewebb** [serge-rider/dbeaver#2565]
After upgrading from v2.4.4 to v2.4.5 columns of Oracle (12c) DATE data type do not correctly show the time portion by default in the Data Grid - they are truncated down to midnight.

Example code:
```
CREATE TABLE lw_tmp (
 id NUMBER NOT NULL,
 last_updated DATE NOT NULL 
);

INSERT INTO lw_tmp 
(id, last_updated) 
VALUES (1, to_date('23/11/2017 13:45:43', 'DD/MM/YYYY HH24:MI:SS'));

SELECT id, last_updated, to_char(last_updated, 'DD/MM/YYYY HH24:MI:SS') AS "Formatted" 
FROM lw_tmp;
```
ID |LAST_UPDATED        |Formatted|
---|--------------------|--------------------|
1  |23/11/2017 00:00:00 |23/11/2017 13:45:43 |

As can be seen the time portion is truncated down to midnight.  

The Pattern set in DBeaver->Result Sets->Data Formats for "Date" is set to "dd/MM/YYYY HH:mm:ss" (see screenshot):

![dbeaver 2 4 5 oracledate](https://user-images.githubusercontent.com/11154505/33156918-62fc89b0-d052-11e7-9d95-26c56f5c35a9.jpg)

This seems similar to an old issue: https://dbeaver.jkiss.org/forum/viewtopic.php?f=2&t=660

Note: this is using the DBeaver Plugin in Eclipse. Relevant package versions are:

|Name|Version|Id|
| ------------- |:-------------:| -----:|
|Eclipse IDE for Java Developers|4.7.1.20171009-0410|epp.package.java|
|DBeaver Core|4.2.5|org.jkiss.dbeaver.core.feature.feature.group|
|DBeaver DB2 Extension|2.1.54|org.jkiss.dbeaver.ext.db2.feature.feature.group|
|DBeaver Generic Extension|2.1.55|org.jkiss.dbeaver.ext.generic.feature.feature.group|
|DBeaver Office Support|1.1.5|org.jkiss.dbeaver.ext.office.feature.feature.group|
|DBeaver Oracle Extension|2.1.54|org.jkiss.dbeaver.ext.oracle.feature.feature.group|
|DBeaver Runtime|4.2.5|org.jkiss.dbeaver.runtime.feature.feature.group|

**Comment by serge-rider** [comment]
This is a bug in 4.2.5 (see #2510), will be fixed in the next version.
Thanks for report anyway.

**Comment by leewebb** [comment]
Confirmed as fixed in v4.2.6

**Comment by davidhooey** [comment]
This is a problem against in 5.2.1.

**Comment by serge-rider** [comment]
@davidhooey duplicate of #4270. Fixed in 5.2.2
